docs/lro-post-must-not-use-original-url-as-final-state.md

```diff
@@ -0,0 +1,13 @@
+# LroPostMustNotUseOriginalUriAsFinalState
+
+## Description
+
+The long running post operation must not use final-stat-via:original-uri.
+
+## How to fix
+
+Change to use other options define in [x-ms-long-running-operation-options](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-long-running-operation-options)
```

Comment:
Consider using states defined in [x-ms-long-running-operation-options](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-long-running-operation-options)